### PR TITLE
Fix argument remapping. Limit remapping to first argument.

### LINKED ISSUE #3295: Drush 9; Argument 'en' is always replaced with 'pm:enable'
If a Drush 9 commands allows arguments, the argument 'en' will be replaced with its full command name pm:enable. In this way it isn't possible to create an argument that accepts the language code 'en'.

Drush 8 doesn't have this issue.

Some test cases:
```Shell
$ drush @site_alias en 'en'
[error]  Unable to install modules pm:enable due to missing modules pm:enable.

$ drush @site_alias pm:enable 'en'
[error]  Unable to install modules pm:enable due to missing modules pm:enable.
```

### PR DESCRIPTION
**Pull request by Sutharsan**
Fixes #3295 by limiting the remapping to the first argument (properly).

### CODE REVIEW

**Comment by weitzman** [comment]
Thanks ... The old code had consistently wrong spelling or no? if yes, then I dont see how a spelling fix can eliminate the bug. I cant find the actual spelling inconsistency except for in Doxygen

**Comment by greg-1-anderson** [comment]
If we're going to call this fixed, it would be helpful to have a failing test that passes when the fix is applied.

**Comment by Sutharsan** [comment]
@weitzman not consistenly wrong, no. See line 50 (old code) that uses correct spelling ` $sawCommand = true;`. Therefore `$sawCommmand` (tripple m) does not get flagged.

**Comment by Sutharsan** [comment]
Minimal unit test added.

File: tests/ArgsRemapperTest.php [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Unish;
+
+use \Drush\Preflight\argsRemapper;
```
Comment (weitzman) [inline tests/ArgsRemapperTest.php]: Capital A in class name, not 'a'.

**Comment by Sutharsan** [comment]
Done

File: tests/ArgsRemapperTest.php (hunk 1 above)
Comment (Sutharsan) [reply tests/ArgsRemapperTest.php]: Done

**Comment by Sutharsan** [comment]
:) Thanks

**Pull request merged**